accesshandler/pkg/providers/aws/sso/access.go

```diff
@@ -172,7 +172,25 @@ func (p *Provider) getUser(ctx context.Context, email string) (*idtypes.User, er
 	return &res.Users[0], nil
 }
 func (p *Provider) Instructions(ctx context.Context, subject string, args []byte) (string, error) {
-	url := fmt.Sprintf("https://%s.awsapps.com/start", p.identityStoreID)
-	instructions := fmt.Sprintf("You can access this role at your [AWS SSO URL](%s)", url)
-	return instructions, nil
+	var a Args
+	err := json.Unmarshal(args, &a)
+	if err != nil {
+		return "", err
+	}
+	po, err := p.client.DescribePermissionSet(ctx, &ssoadmin.DescribePermissionSetInput{
+		InstanceArn: aws.String(p.instanceARN.Get()), PermissionSetArn: aws.String(a.PermissionSetARN),
+	})
+	if err != nil {
+		return "", err
+	}
+	url := fmt.Sprintf("https://%s.awsapps.com/start", p.identityStoreID.Get())
+
+	i := "# Browser\n"
+	i += fmt.Sprintf("You can access this role at your [AWS SSO URL](%s)\n\n", url)
+	i += "# CLI\n"
+	i += "Ensure that you've [installed](https://docs.commonfate.io/granted/getting-started#installing-the-cli) the Granted CLI, then run:\n\n"
+	i += "```\n"
+	i += fmt.Sprintf("assume --sso --sso-start-url %s --sso-region %s --account-id %s --role-name %s\n", url, p.region.Get(), a.AccountID, aws.ToString(po.PermissionSet.Name))
+	i += "```\n"
```

Comment:
Love this feature/concept

Having a thought here: we might want to consider perma-links for features like this? ensure links stick around, are robust, not easily deleted or have their URLs changed. Low risk, but worth some thought